snafu-derive/src/lib.rs

```diff
@@ -932,17 +916,17 @@ impl syn::parse::Parse for Backtrace {
 
         if lookahead.peek(LitBool) {
             let val: LitBool = input.parse()?;
-            Ok(Backtrace::Flag(val.value))
+            Ok(Backtrace(val.value))
         } else if lookahead.peek(Ident) {
             let name: Ident = input.parse()?;
 
             if name == "delegate" {
-                Ok(Backtrace::Delegate)
-            } else {
                 Err(syn::Error::new(
                     name.span(),
-                    "expected `true`, `false`, or `delegate`",
+                    "`backtrace(delegate)` has been removed; use `backtrace` on a source field",
```

Comment:
I like this kind of helpful text, but I also would only want to keep it for "so long". I wonder if there's a way to indicate that this code can be removed at some future point (that I don't know what it would be)

Reply:
I agree, but I didn't have a great idea other than perhaps adding a comment.  "Removal message added 2019-08-07, delete when people have forgotten about delegate" or something :)

Or perhaps as one of the line items for a 1.0 release?

Reply:
It's probably not important either way. Let's just leave it until it gets in the way and can then evaluate.